app.py

```diff
@@ -207,6 +209,28 @@ def save_identity():
     return jsonify({})
 
 
+# End Point example which verifies payload signature
+@app.route('/v1/signed_payload', methods=['POST'])
+@validate_json
+def signed_payload():
+    payload = request.get_json(force=True)
+    signature_base64_encoded = request.headers.get('signature')
+    identity = payload['identity']
+
+    signature_bytes = base64.b64decode(signature_base64_encoded)
+
+    recovered_public_address = recover_public_address(
+        request.data,
+        signature_bytes,
+    )
+
+    # verify payload
+    if recovered_public_address.lower() == identity.lower():
```

Comment:
Is it OK to lower, I am for case sensitive

Reply:
the main question is for geth, if it is case insensitive, we don't have much options if we want to be compatible

Reply:
after discussion:
- Rule1: user input -> Sanitize input -> Database write
- Rule2: user input -> Sanitize input -> Database search

Reply:
identity in lower case will be written in database